web/templates/admin/invite-list.tmpl

```diff
@@ -84,10 +99,16 @@
                 >{{$creator}}</span>, {{human_time .CreatedAt}}
             {{end}}
           </span>
-          <a
-            href="{{urlTo "admin:invites:revoke:confirm" "id" .ID}}"
-            class="pl-4 w-20 py-2 text-center text-gray-400 hover:text-red-600 font-bold cursor-pointer"
-            >{{i18n "AdminInviteRevoke"}}</a>
+          {{ if or member_is_elevated $hasCreatedInvite }}
+            <a
+              href="{{urlTo "admin:invites:revoke:confirm" "id" .ID}}"
+              class="pl-4 w-20 py-2 text-center text-gray-400 hover:text-red-600 font-bold cursor-pointer"
+              >{{i18n "AdminInviteRevoke"}}</a>
+          {{ else }}
+            <span 
+            class="pl-4 w-20 py-2 text-center text-gray-400 line-through font-italic cursor-not-allowed"
+            >Revoke</span>
```

Comment:
What if show nothing?

Reply:
Then we have an empty column named "Action", thought it was a bit jank & better to show the action that can't be taken. It's up to you

Reply:
Might be visually more pleasing to have emptiness (a strike-through is visually busy and not so easy to read, and could be repeated potentially 20x), so let's try the nothingness

Reply:
![image](https://user-images.githubusercontent.com/3862362/115569630-b5f23480-a2bd-11eb-9dd7-7fffc9a40d3a.png)

Reply:
My life for Aiur